app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class AddressSearchController < ApplicationController
+      include ArcgisApi
+
+      def geocoder
+        @geocoder ||= Geocoder.new
+      end
+
+      def suggest(address)
+        geocoder.suggest(address)
+      end
+
+      def find_address_candidates(magic_key)
+        geocoder.find_address_candidates(magic_key)
+      end
+
+      def index
+        arcgis_api_response = []
+        begin
+          suggestion = suggest(permitted_params[:address]).first
+          arcgis_api_response = [find_address_candidates(suggestion.magic_key).first]
+        rescue Faraday::ConnectionFailed => _error
+          nil
+        end
+
+        render json: arcgis_api_response.to_json
```

Comment:
the `.to_json` is redundant, `render json:` takes the array, hash, etc and serializes it